compiler/src/generate/t_ruby_generator.cc

```diff
@@ -777,6 +780,19 @@ void t_ruby_generator::generate_writer(t_ruby_ofstream& out, t_struct* tstruct)
   out.indent() << "end" << endl << endl;
 }
 
+void t_ruby_generator::generate_schema_annotations(t_ruby_ofstream& out, t_struct* tstruct) {
+  std::map<string, string> ann = tstruct->annotations_;
```

Comment:
You are not update anything in `ann` ie. `tstruct->annotations_`, right? If no, you can use reference to avoid creating a map here. Like `std::map<string, string>& ann = tstruct->annotations_;`

Reply:
Actually, you can just use it directly.